userguide/conf.py

```diff
@@ -116,6 +116,7 @@
 .. |ui-jail-start|        replace::  (Start)
 .. |ui-jail-stop|         replace::  (Stop)
 .. |ui-jail-update|       replace::  (Update)
+.. |ui-jail-shell|        replace:: 塞 SHELL
```

Comment:
Should this be all caps?  Even if the button is, the label here is meant to explain what the icon does.

Reply:
Yes, this is meant to be in all caps, and the label is correct. If you log in to one of the testing systems and look at the option in question, it is identical to how the replacement is configured.